examples/Atlas/+atlasParams/Base.m

```diff
@@ -28,7 +28,7 @@
                                                       'max', [100;100;100;50;50;50]),...
                                'weight', num2cell(zeros(1, nbod)),...
                                'Kd', mat2cell(zeros(6, nbod), 6, ones(1, nbod)));
-      obj.body_motion(r.findLinkId('pelvis')).weight = 0.1;
+      obj.body_motion(r.findLinkId('pelvis')).weight = 0.01;
```

Comment:
ah.  i remember now that this is an optimization "weight" and has nothing to do with the mass of the pelvis.